src/components/appNavigation/TabsView.js

```diff
@@ -1,21 +1,23 @@
 //@flow
 import React from 'react'
 import { View, Text, StyleSheet, Image } from 'react-native'
+import { normalize } from 'react-native-elements'
+import burgerIcon from '../../assets/burgerIcon.png'
+import { toggleSidemenu } from '../../lib/undux/utils/sidemenu'
+import GDStore from '../../lib/undux/GDStore'
 
 type TabButtonProps = {
   text?: string,
   routeName: string,
   icon: string,
-  goTo: (routeKey: string) => void,
-  iconStyle: any,
-  tabButtonStyle: any
+  goTo: (routeKey: string) => void
 }
 
 const TabButton = (props: TabButtonProps) => {
```

Comment:
Why don't create another component? This is not longer a TabMenu

Reply:
Not sure...now we don't have rewards, buy sell and Donate, but the menu is still there...I think we remove those menu items because it's the first version, but the platform will have them later